src/payment/strategies/amazon-maxo/amazon-maxo-script-loader.ts

```diff
@@ -0,0 +1,32 @@
+import { ScriptLoader } from '@bigcommerce/script-loader';
+
+import { PaymentMethodClientUnavailableError } from '../../errors';
+import PaymentMethod from '../../payment-method';
+
+import { AmazonMaxoHostWindow, AmazonMaxoRegions, AmazonMaxoSDK } from './amazon-maxo';
+
+export default class AmazonMaxoScriptLoader {
+    constructor(
+        private _scriptLoader: ScriptLoader,
+        private _window: AmazonMaxoHostWindow = window
+    ) {}
+
+    // verify the url depending to the zone
+    load(method: PaymentMethod): Promise<AmazonMaxoSDK> {
+        const {
+            initializationData: { region = 'us' } = {},
+        } = method;
+
+        const amazonMaxoRegion = (AmazonMaxoRegions as any)[region];
```

Comment:
Why the casting to any?

Reply:
Fixed ♻️

Reply:
`region` is destructured from the `method`'s field `initializationData`, which is defined as `any` on the PaymentMethod interface, this causes Typescript to understand region as a probable `any` value, but I'm defining a default value on line 17.

This causes the need to cast the amazonMaxoRegion Enum as any, in order to get the proper value for the region